admin_manual/installation/system_requirements.rst

```diff
@@ -27,22 +28,21 @@ Server
 +------------------+-----------------------------------------------------------------------+
 | Web server       | - Apache 2.4 with ``prefork`` :ref:`apache-mpm-label` and ``mod_php`` |
 +------------------+-----------------------------------------------------------------------+
-| PHP Runtime      | - 5.6+, 7.0, & 7.1                                                    |
+| PHP Runtime      | - 7.1+                                                                |
```

Comment:
@settermjd have we finally got to the point where we do not support PHP 5.6 (or 7.0)?
They have security fixes for the next few months still, and I have no idea how many people are on those PHP  versions until the last moment.

Reply:
See comment below on the commit 🙈  - we are still support php5.6 - it is however recommended to go to 7.1+

Reply:
It's likely that I changed this by mistake, as I referred to composer.json in master, which contains `"php": ">=7.1",`.

Reply:
@patrickjahns, regarding [your other comment](https://github.com/owncloud/documentation/pull/4414#commitcomment-30451429), I'll revert this change.